src/codegen/capacity/estimators.ts

```diff
@@ -0,0 +1,173 @@
+import {JsExpression} from '@jsonjoy.com/util/lib/codegen/util/JsExpression';
+import {MaxEncodingOverhead, maxEncodingCapacity} from '@jsonjoy.com/util/lib/json-size';
+import type {CapacityEstimatorCodegenContext} from './CapacityEstimatorCodegenContext';
+import type {Type} from '../../type';
+
+type EstimatorFunction = (ctx: CapacityEstimatorCodegenContext, value: JsExpression, type: Type) => void;
+
+function normalizeAccessor(key: string): string {
```

Comment:
Use *ArrowFunction*s throughout this file for standalone functions.

```suggestion
const normalizeAccessor = (key: string): string => {
```